apps/docs/publisher_agent.md

```diff
@@ -9,7 +9,7 @@ To run the agent on your infrastructure, you'll need to first create a `config.j
 
 Here we open a websocket at `ws://localhost:5216/publish` to receive price updates. The agent will sign and send the latest price every 500 ms (clock updates) or when the price has changed by more than 0.1% (delta updates). 
 
-`config.json` follows the structure of the [ConfigFile struct](config.go):
+`config.json` follows the structure of the [ConfigFile struct](../lib/publisher_agent/config.go#L27):
```

Comment:
ah good catch. Let's leave out the line number though - otherwise we'll have to keep updating the README whenever we change this go file